core-bundle/src/Resources/contao/library/Contao/InsertTags.php

```diff
@@ -730,7 +730,7 @@ static function ($allowedTag)
 							break;
 
 						case 'path':
-							$arrCache[$strTag] = Idna::decode(Environment::get('base'));
+							$arrCache[$strTag] = Idna::decode(Environment::get('path')) . '/';
```

Comment:
I would prefer to document the change and fix the `{{env::path}}` insert tag in Contao 5.

Reply:
I dug a bit in the code history, `{{env::path}}` was referring to `get('base')` ever since: https://github.com/contao/core/blob/4a2809d34ce08193c26b1c5815ecad2c1cdb909b/system/libraries/Controller.php#L1303-L1305 (Initial Commit)

I think we should do the following:
1. Deprecate `{{env::path}}` but keep the functionality as it is currently
2. Add `{{env::base}}` as the replacement
3. Add `{{env::base_path}}` (as a counterpart to `{{env::base_url}}`) and use the prefix `{{env::base_path}}/` everywhere.

This would streamline the naming to be straight forwad IMO:
1. `{{env::base}}` refers to `get('base')`
2. `{{env::base_path}}` refers to `$request->getBasePath()`
3. `{{env::base_url}}` refers to `$request->getBaseUrl()` (as it already does)

Reply:
Changed in 56aa6a53b57d6db22ab782ffac95d8a2e5d2effe.